packages/types/src/span.ts

```diff
@@ -35,10 +35,21 @@ export interface Span {
    */
   setHttpStatus(httpStatus: number): this;
 
+  /**
+   * Creates a new `Span` while setting the current `Span.id` as `parentSpanId`.
+   * Also the `sampled` decision will be inherited.
+   */
+  child(spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId'>>): Span;
```

Comment:
Could you update this to exclude more keys?

```js
  public child(spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId'>>): Span {
    const span = new Span({
      ...spanContext,
      parentSpanId: this._spanId,
      sampled: this.sampled,
      traceId: this._traceId,
    });
```

`parentSpanId`, `sampled` and `traceId` should all be invalid, right?

Needs to be changed in the implementation of `child` as well.